crates/sui-sdk/src/apis.rs

```diff
@@ -1173,7 +1173,13 @@ impl QuorumDriverApi {
         }
 
         // JSON-RPC ignores WaitForLocalExecution, so simulate it by polling for the transaction.
-        let mut poll_response = tokio::time::timeout(WAIT_FOR_LOCAL_EXECUTION_TIMEOUT, async {
+        let wait_for_local_execution_timeout: Duration = if cfg!(msim) {
```

Comment:
Can you not leave it as a constant by doing something like this?

```
#[cfg(not(msim))]
const WAIT_FOR_LOCAL_EXECUTION_TIMEOUT: Duration = Duration::from_secs(60);
#[cfg(msim)]
const WAIT_FOR_LOCAL_EXECUTION_TIMEOUT: Duration = Duration::from_secs(120);
```

Reply:
That would be compile time const, which doesn't work because the simtests we want to affect are in other crates.